docs/src/components/home/DesignKits.tsx

```diff
@@ -259,10 +259,11 @@ export default function DesignKits() {
           ...theme.applyDarkStyles({
             background: `linear-gradient(to bottom, ${
               (theme.vars || theme).palette.primaryDark[900]
-            } 0%, ${alpha((theme.vars || theme).palette.primaryDark[900], 0)} 30%, ${alpha(
-              (theme.vars || theme).palette.primaryDark[900],
-              0,
-            )} 70%, ${(theme.vars || theme).palette.primaryDark[900]} 100%)`,
```

Comment:
I wonder what is the alternative for `alpha`/`lighten`/`darken` utility functions when using CSS variables?
For `alpha`, something like this should work, right?
```
'rgba(${theme.vars.palette.primary.mainChannel} / 0.5'
```
But then, not every color in the palette has a `xxxChannel` representation.

Reply:
I believe we observed a similar problem in https://github.com/mui/mui-x/pull/6784

Reply:
cc @siriwatknp

Reply:
`'rgba(${theme.vars.palette.primary.mainChannel} / 0.5'` this is correct, and yes, we are generating channels for some colors not all.

Reply:
What about this case?
```js
'rgba(${theme.vars.palette.primaryDark[900]} / 0.5)'
```
The above won't work, and there's no channel for `primaryDark[900]` color.

Reply:
> rgba(${theme.vars.palette.primaryDark[900]} / 0.5)

Where is it? The above looks fine to me.